### PR TITLE
FEAT: synchronization IP-address for nameservers

### PR DESCRIPTION
**Pull request by EwaldvanGeffen**
Changes proposed in this pull request:
- add synchronization ip field for nameserver to use for rsync purposes. I prefer to have my communication run over a different IP-address than the public name-server IP-address that's now being employed.

This PR is for reference purposes; the commit history got a bit a mess aswell as the code. Please refactor as you see fit. I think it would be better if Export/Engine would use a function to fetch {address} or {syncaddress} instead of the hardcoded parameter now.

I run this in production and it works, albeit dirty :)

### CODE REVIEW

**Comment by msimerson** [comment]
This is entirely redundant. The IP addresses in the nameserver definition **is** the "sync" IP address used for rsync. Set it to whatever IP address you use for rsync, or set it as the NS public IP and then edit the IPs in the `Makefile` generated when you set up the export, and that's the end of the matter.

### TL;DR

The Nameservers assigned to a zone put these records inside a zone: (example: simerson.net):

````
simerson.net.   14400   IN      NS      ns1.cadillac.net.
simerson.net.   14400   IN      NS      ns2.cadillac.net.
simerson.net.   14400   IN      NS      ns3.cadillac.net.
simerson.net.   14400   IN      NS      ns4.cadillac.net.
````

The IP addresses for those NS records are defined in the cadillac.net zone:

````
% grep ^ns cadillac.net 
ns1	86400	IN  A	138.210.133.61
ns2	86400	IN  A	192.48.85.146
ns2	86400	IN  AAAA	2607:f060:b008:feed:0000:0000:0000:0006
ns3	86400	IN  A	46.105.8.147
ns3	86400	IN  A	66.128.51.174
ns3	86400	IN  AAAA	2605:ae00:0329:0000:0000:0000:0000:000f
ns3	86400	IN  MX	10	ns3.cadillac.net.
ns4	3600	IN  A	204.13.250.12
ns4	3600	IN  A	204.13.251.12
ns4	3600	IN  A	208.78.70.12
ns4	3600	IN  A	208.78.71.12
ns4	3600	IN  AAAA	2001:0500:0090:0001:0000:0000:0000:0012
ns4	3600	IN  AAAA	2001:0500:0094:0001:0000:0000:0000:0012
````
